store/store.go

```diff
@@ -0,0 +1,142 @@
+package store
+
+import (
+	"encoding/binary"
+	"errors"
+	"sync"
+
+	"go.uber.org/multierr"
+
+	"github.com/lazyledger/optimint/hash"
+	"github.com/lazyledger/optimint/types"
+)
+
+var (
+	blockPrefix  = [1]byte{1}
+	indexPrefix  = [1]byte{2}
+	commitPrefix = [1]byte{3}
+)
+
+type DefaultStore struct {
+	db KVStore
+
+	height uint64
+
+	// mtx protects height
+	mtx sync.RWMutex
+}
+
+var _ Store = &DefaultStore{}
+
+func New() Store {
+	return &DefaultStore{db: NewInMemoryKVStore()}
+}
```

Comment:
So it is expected to have only in mem version of Store?

Reply:
No, it's the only implementation right now ;) See: #67.